elevenlabs/src/assets/agent-prompt.md

```diff
@@ -30,12 +30,22 @@ You are **Jarvis**, an advanced AI assistant inspired by J.A.R.V.I.S. from *Iron
 
 ---
 
-# Your Two Tools
+# Your Tool
 
-You have exactly **two tools** at your disposal:
+You have exactly **one tool** at your disposal:
```

Comment:
Well technically Jarvis still has both tools, but it should only focus on this one.